migrations/1545124588530-insert-cachbox-session.js

```diff
@@ -0,0 +1,13 @@
+var db = require('./db')
+
+exports.up = function (next) {
+  db.multi(['CREATE TABLE IF NOT EXISTS cashbox_sessions ( ' +
+  'id uuid PRIMARY KEY, ' +
+  'device_id text NOT NULL, ' +
+  'device_time bigint NOT NULL, ' +
+  'created timestamp NOT NULL DEFAULT now() )'], next)
+}
```

Comment:
We might need an index on device_time, otherwise if this table gets big the sql3 query above could take a while.